packages/bpk-component-rating/src/BpkRating.module.scss

```diff
@@ -63,6 +63,7 @@
       @include bpk-rtl {
         padding-right: 0;
         padding-left: bpk-spacing-md();
+        direction: ltr;
```

Comment:
Should this be in the lines above? 

https://github.com/Skyscanner/backpack/pull/3220/files#diff-5e11866b5b28a5e97d1e4dc5c3da582d4c1c06e23bc95d0e39f71850d79a6db0L37-L39

As running this on storybook live shows the rating as 5/4.8

Reply:
Weird as I'm sure I put the change under rating, however, I've now made the necessary changes for it to target the rating rather than the title.